DEVELOPMENT.md

```diff
@@ -18,9 +18,11 @@ You need to install:
 - [`ko`](https://github.com/google/ko) - (_required_)
 - [`docker`](https://www.docker.com/) - (_required_)
 - [`Go`](https://golang.org/) - (_required_)
+  - check [go <version>](https://github.com/knative-sandbox/eventing-kafka-broker/blob/master/go.mod) for the required Go version used in this project
```

Comment:
GitHub shows only `go` instead of `go <version>`, see https://github.com/knative-sandbox/eventing-kafka-broker/pull/27/files?short_path=f785e9e#diff-f785e9e46b9afd2567df7a658338db17

Reply:
sorry for the multiple force push.. i preview the changes on my local MD preview on vscode, it works fine but not on GitHub. so i had to be pushing first to see if it works. can you take a look again?

Reply:
No problem! :+1: